Packs/Grafana/Integrations/Grafana/Grafana.py

```diff
@@ -0,0 +1,799 @@
+    )
+
+    return command_results
+
+
+def org_get_by_id_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    org_id = args.get('org_id')
+
+    response = client.org_get_by_id_request(org_id)
+    command_results = CommandResults(
+        outputs_prefix='Grafana.Organization',
+        outputs_key_field='id',
+        outputs=response,
+        raw_response=response,
+        readable_output=tableToMarkdown(f'Organization {org_id} Results', response, removeNull=True,
+                                        headerTransform=pascalToSpace)
+    )
+
+    return command_results
+
+
+def dashboards_search_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    query = args.get('query')
+    tag = argToList(args.get('tag', ''))
+    type_ = args.get('type')
+    dashboard_ids = argToList(args.get('dashboard_ids', ''))
+    folder_ids = argToList(args.get('folder_ids', ''))
+    starred = args.get('starred')
+    limit = args.get('limit')
+    page = args.get('page')
+
+    response = client.dashboards_search_request(query, tag, type_, dashboard_ids, folder_ids, starred, limit, page)
+    command_results = CommandResults(
+        outputs_prefix='Grafana.Dashboard',
+        outputs_key_field='id',
+        outputs=response,
+        raw_response=response,
+        readable_output=tableToMarkdown('Dashboards', response, removeNull=True, headerTransform=pascalToSpace)
+    )
+
+    return command_results
+
+
+def test_module(client: Client, params: dict) -> None:
+    message: str = ''
+    try:
+        if client.users_search_request():
+            message = 'ok'
+
+        if params['isFetch']:
+            max_fetch = arg_to_number(params.get('max_fetch'))
+            if max_fetch and (max_fetch > MAX_INCIDENTS_TO_FETCH or max_fetch <= 0):
+                raise DemistoException(f'Maximum number of incidents to fetch exceeds the limit '
+                                       f'(restricted to {MAX_INCIDENTS_TO_FETCH})')
+            query = params.get('query')
+            if query:
+                query = url_encode(query)
+            client.alerts_list_request(dashboard_id=argToList(params.get('dashboard_id')), panel_id=params.get('panel_id'),
+                                       query=query, state=params.get('state'))
+
+    except DemistoException as e:
+        if 'Unauthorized' in str(e):
+            message = 'Authorization Error: make sure username and password are correctly set'
+        else:
+            raise e
+    return return_results(message)
+
+
+def fetch_incidents(client: Client, first_fetch: str, dashboard_id: str = None, panel_id: str = None,
+                    alert_name: str = None, state: str = None, limit: Optional[int] = MAX_INCIDENTS_TO_FETCH) -> List[dict]:
+    if not limit:
+        limit = MAX_INCIDENTS_TO_FETCH
+    last_fetch = demisto.getLastRun().get('last_fetch', None)
```

Comment:
`None` should not be a specified default value, as it is the default default :)
```suggestion
    last_fetch = demisto.getLastRun().get('last_fetch')
```